src/mavedb/lib/score_sets.py

```diff
@@ -127,7 +131,50 @@ def search_score_sets(db: Session, owner: Optional[User], search: ScoreSetsSearc
         )
 
     score_sets: list[ScoreSet] = (
-        query.join(ScoreSet.experiment).join(ScoreSet.target_genes).order_by(Experiment.title).all()
+        query.join(ScoreSet.experiment)
```

Comment:
is it intentional that the top-level `.join(ScoreSet.target_genes)` goes away here?